{
    "ID": {
        "GitHub_Advisory_Database": {
            "aliases": [
                "CVE-2023-47102"
            ]
        },
        "Nomisec": {
            "id": 720393369,
            "name": "CVE-2023-47102",
            "full_name": "nitipoom-jar/CVE-2023-47102"
        }
    },
    "summary": null,
    "details": {
        "GitHub_Advisory_Database": "UrBackup Server 2.5.31 allows brute-force enumeration of user accounts because a failure message confirms that a username is not valid."
    },
    "description": {
        "Nomisec": null
    },
    "topics": [
        "enumeration",
        "accounts",
        "31",
        "failure",
        "force",
        "valid",
        "username",
        "message",
        "server",
        "user"
    ],
    "severity": [],
    "affected": [],
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://nvd.nist.gov/vuln/detail/CVE-2023-47102"
        },
        {
            "type": "WEB",
            "url": "https://quantiano.github.io/cve-2023-47102"
        }
    ],
    "proof_of_concept": {
        "owner": {
            "html_url": "https://github.com/nitipoom-jar"
        },
        "html_url": "https://github.com/nitipoom-jar/CVE-2023-47102"
    },
    "has_discussions": null,
    "database_specific": {
        "cwe_ids": [
            "CWE-203"
        ],
        "severity": "MODERATE",
        "github_reviewed": false,
        "github_reviewed_at": null,
        "nvd_published_at": "2023-11-07T06:15:12Z"
    },
    "Time_Description": {
        "GitHub_Advisory_Database": {
            "modified": "2023-11-13T21:30:57Z",
            "published": "2023-11-13T21:30:57Z",
            "withdrawn": null
        },
        "Nomisec": {
            "created_at": "2023-11-18T10:52:40Z",
            "updated_at": "2023-11-18T12:30:55Z",
            "pushed_at": "2023-11-18T12:39:52Z"
        }
    }
}